tsconfig.base.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "strict": true,
+    "target": "ES2022",
+
+    // TODO: Over time, enable these strict options
+    "noImplicitAny": false,
+    "strictNullChecks": false,
+    "useUnknownInCatchVariables": false
```

Comment:
[Explanation] `strict` is generally a best practice, and has flags added to it over time. For anything that can't be added yet, the common practice is to disable them specifically with a TODO comment. Kind of like enabling a recommended preset in an ESLint config, with `rules` overrides.

I know there was mention of not wanting some or all of these - is here a good place to discuss?

Reply:
I think strict vs. not should be opened up as a separate discussion.

Reply:
👍 #118